aslam_cv/aslam_cv_error_terms/include/aslam/backend/implementation/CovarianceReprojectionError.hpp

```diff
@@ -70,14 +76,14 @@ namespace aslam {
 
     	A(0,1) += J(0);
```

Comment:
Ok, this one is somewhat speculative, but is the naming of the method misleading? After all, A does not seem to be a covariance matrix. Or did I get something wrong?